content/docs/concepts/performance-and-scalability/index.md

```diff
@@ -13,127 +13,110 @@ aliases:
 keywords: [performance,scalability,scale,benchmarks]
 ---
 
-We follow a four-pronged approach to Istio performance characterization, tracking and improvements:
+Istio makes it easy to create a network of deployed services with rich routing,
+load balancing, service-to-service authentication, monitoring, and more - all
+without any changes to the application code. Istio strives to provide
+these benefits with minimal resource overhead and aims to support very
+large meshes with high request rates while adding minimal latency.
 
-* Code level micro-benchmarks
+The Istio data plane components, the Envoy proxies and Mixer for headers only, handle
+data flowing through the system. The Istio control plane components, Pilot, Galley and Citadel,
+configure the data plane. The data plane and control plane have distinct
+performance concerns.
 
-* Synthetic end-to-end benchmarks across various scenarios
+## Data plane performance
 
-* Realistic complex app end-to-end benchmarks across various settings
+Data plane performance depends on many factors, for example:
 
-* Automation to ensure performance doesn't regress
+- Number of client connections
+- Target request rate
+- Request size and Response size
+- Enabling `mTLS`
+- Number of proxy worker threads
+- Protocol
+- CPU cores
```

Comment:
Is CPU cores a factor? Of course, the more cores you have, the better the performance, but this is a generic thing.  Shouldn't you mention mixer filter enabled/disabled and number of  policies as factors.